crowbar_framework/app/models/backup.rb

```diff
@@ -100,13 +100,21 @@ def create_archive
 
     Crowbar::Backup::Export.new(dir).export
     Dir.chdir(dir) do
-      ar = Archive::Compress.new(
+      cmd = [
+        "tar",
+        "--same-permissions",
+        "--same-owner",
```

Comment:
the problem I thought started with trying to pack files that don't belong to crowbar. I assumed libarchive does not have the permissions to pack them, that's why I used a system call (which in return makes me wonder why I haven't used sudo here...).
I guess I was wrong. It seems we can keep the code here as it was, I will have to test it